sw/device/boot_rom/BUILD

```diff
@@ -0,0 +1,73 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU", "opentitan_binary")
+load("//rules:autogen.bzl", "autogen_chip_info")
+
+autogen_chip_info(
+    name = "chip_info",
+)
+
+opentitan_binary(
+    name = "boot_rom",
+    output_scrambled = True,
+    deps = [
+        ":boot_rom_lib",
+        "//sw/device/lib/base:mmio",
+    ],
+)
+
+cc_library(
+    name = "boot_rom_lib",
+    srcs = [
+        "boot_rom.c",
+    ],
+    linkopts = [
+        "-T $(location rom_link.ld)",
```

Comment:
If this is just a library, why does it have a linker script on it? Seems suspect.

Reply:
I think the behavior and the intent is that it gets passed along the dependency chain to where we need it. I see now how that conflicts with the intent of the linker script and I'm happy to move it to the opentitan_binary target to reflect that.